consensus/istanbul/backend/engine_test.go

```diff
@@ -249,7 +249,7 @@ func makeHeader(parent *types.Block, config *istanbul.Config) *types.Header {
 	header := &types.Header{
 		ParentHash: parent.Hash(),
 		Number:     parent.Number().Add(parent.Number(), common.Big1),
-		GasLimit:   core.CalcGasLimit(parent, parent.GasLimit(), parent.GasLimit()),
+		GasLimit:   core.CalcGasLimit(parent, nil, parent.GasLimit(), parent.GasLimit()),
```

Comment:
Do we need to store this in the block header at all anymore, since it can be read from the state trie?

Reply:
Well there is still the time before the contracts are deployed, how should that be handled?

Reply:
Good question. Should we have a fallback value that we define in protocol_params.go?